### DISCUSSION TITLE
"Not found" for any package I try to install

### DISCUSSION THREAD

**Discussion by celiopy** [LingmoOS/Community#24] [Bugs Report/缺陷提交] [answered]
Hello, when I try to update or install a package I get [Not found] as if the package isn't in the mirror.

`Err:1 https://packages.lingmo.org/lingmo hydrogen/main amd64 cups-pk-helper amd64 0.2.6-2
  404  Not Found [IP: 443]
Err:2 https://packages.lingmo.org/lingmo hydrogen/main amd64 hyperv-daemons amd64 6.11-1
  404  Not Found [IP:  443]
Err:3 https://packages.lingmo.org/lingmo hydrogen/main amd64 linux-libc-dev all 6.11-1
`

In the example, I tried to update the system.

[celio@los][~]$ sudo apt search chromium
[...]
chromium/hydrogen 128.0.6613.137-1 amd64
  web browser
[...]

It finds the package I look for in the repo.

```
[celio@los][~]$ sudo apt install chromium
Installing:                                 
  chromium
[...]
Continue? [S/n] 
Err:1 https://packages.lingmo.org/lingmo hydrogen/main amd64 chromium-common amd64 128.0.6613.137-1
  404  Not Found [IP: 443]
Err:2 https://packages.lingmo.org/lingmo hydrogen/main amd64 chromium amd64 128.0.6613.137-1
  404  Not Found [IP: 443]
Err:3 https://packages.lingmo.org/lingmo hydrogen/main amd64 chromium-sandbox amd64 128.0.6613.137-1
  404  Not Found [IP: 443]
```

It tries to install but states that it can not be found...

**Comment by samwhelp** [comment]
> the same situation occurs on my system ([Lingmo OS 2.0 Hydrogen](https://github.com/orgs/LingmoOS/discussions/19)).




## Info

run to view `/etc/apt/sources.list`.

``` sh
cat /etc/apt/sources.list
```

show

```
# Lingmo OS apt Sources list
deb http://packages.lingmo.org/lingmo hydrogen main non-free non-free-firmware
# deb-src http://packages.lingmo.org/lingmo hydrogen main non-free non-free-firmware

# Lingmo OS Team
```


run

```
sudo apt-get update
```

show

```
Hit:1 http://packages.lingmo.org/lingmo hydrogen InRelease
Reading package lists... Done
```

run to install package `hello`.

``` sh
sudo apt-get install hello
```

show

```
[sudo] password for user:
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  hello
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 53.1 kB of archives.
After this operation, 284 kB of additional disk space will be used.
Err:1 http://packages.lingmo.org/lingmo hydrogen/main amd64 hello amd64 2.10-3
  404  Not Found [IP: 104.21.91.161 80]
E: Failed to fetch http://packages.lingmo.org/lingmo/pool/main/h/hello/hello_2.10-3_amd64.deb  404  Not Found [IP: 104.21.91.161 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

try download [http://packages.lingmo.org/lingmo/pool/main/h/hello/hello_2.10-3_amd64.deb](http://packages.lingmo.org/lingmo/pool/main/h/hello/hello_2.10-3_amd64.deb)

``` sh
wget http://packages.lingmo.org/lingmo/pool/main/h/hello/hello_2.10-3_amd64.deb
```

show

```
--2024-10-07 16:34:54--  http://packages.lingmo.org/lingmo/pool/main/h/hello/hello_2.10-3_amd64.deb
Resolving packages.lingmo.org (packages.lingmo.org)... 104.21.91.161, 172.67.175.90, 2606:4700:3035::ac43:af5a, ...
Connecting to packages.lingmo.org (packages.lingmo.org)|104.21.91.161|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-10-07 16:34:55 ERROR 404: Not Found.
```


using firefox to view page [http://packages.lingmo.org/lingmo/pool/main/](http://packages.lingmo.org/lingmo/pool/main/)

show

```
Index of /lingmo/pool/main/

../
a/                                                 07-Oct-2024 03:32       -
b/                                                 07-Oct-2024 13:50       -
l/                                                 07-Oct-2024 03:33       -
n/                                                 07-Oct-2024 03:32       -
o/                                                 07-Oct-2024 03:31       -
s/                                                 07-Oct-2024 03:31       -
```

> not all debian package in the web site ([http://packages.lingmo.org/lingmo](http://packages.lingmo.org/lingmo)).

**Comment by chun-awa** [comment, marked as answer]
run `sudo apt upgrade base-files` and rewrite sources.list (choose Y)
![1728340616375](https://github.com/user-attachments/assets/113782d7-5d0d-49bf-b73f-477ba260146d)

**Comment by celiopy** [comment]
I see... Just so I know, the upcoming release, the real major release, that problem won't happen, right?

**Reply by chun-awa** [reply]
yes
